lua/telescope/builtin/init.lua

```diff
@@ -316,10 +316,12 @@ builtin.lsp_references = require("telescope.builtin.lsp").references
 
 --- Goto the definition of the word under the cursor, if there's only one, otherwise show all options in Telescope
 ---@param opts table: options to pass to the picker
+---@field jump_type string: how to go to definition if there is only one, values: "tab", "split", "vsplit", "never"
 builtin.lsp_definitions = require("telescope.builtin.lsp").definitions
 
 --- Goto the implementation of the word under the cursor if there's only one, otherwise show all options in Telescope
 ---@param opts table: options to pass to the picker
+---@field jump_type string: how to go to definition if there is only one, values: "tab", "split", "vsplit", "never"
```

Comment:
There is typo in this line, but fixing this will make the line too long.

Reply:
i see

Reply:
Oh, it seems it won't make this line too long.

Reply:
Also, due to the line length limit, I couldn't add how to keep the original behavior to the documentation comments.

Reply:
you meant that right

Reply:
> Also, due to the line length limit, I couldn't add how to keep the original behavior to the documentation comments.

Just not define it. Yeah its currently a limitation we have to live with. Ill merge. Thanks for the PR :)